src/Controllers/SoundingBoardController.php

```
@@ -261,11 +261,7 @@ public function respond($id): void
 
             // Update the individual response status
             $results[$memberIndex]['status'] = $action . 'ed';
-            $this->db->query("UPDATE evaluation_results SET results_json = :results_json WHERE id = :id", [
-                ':results_json' => json_encode($results),
-                ':id'           => $id,
-            ]);
-
+            EvaluationResult::updateResultsJson($this->db, (int) $id, $orgId, (string) json_encode($results));
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\StratFlow\Models\EvaluationResult' in method 'respond'. (undefined)

(StaticAccess)